components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/editor/js/design-view/form-builder/form-builder.js

```diff
@@ -354,6 +354,92 @@ define(['require', 'log', 'jquery', 'lodash', 'formUtils', 'streamForm', 'tableF
             partitionForm.generatePropertiesForm(element, formConsole, formContainer);
         };
 
+        FormBuilder.prototype.GeneratePropertiesFormForStreamWizard = function (formContainer, _uniqueStreamId, streamObject) {
+            var self = this;
+
+            var formOptions = {};
```

Comment:
This can be simply achieved as,

```js
var formOptions = {
    configurationData: self.configurationData,
    application: self.application,
   ...
};
```